armbian/mender-convert.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
-# BitBoxBase: Create Mender-enabled images
+# BitBox Base: Create Mender-enabled images
 #
 # Script to automate the conversion process of an Armbian image into a
-# Mender provisioning image and update artefacts for the BitBoxBase.
+# Mender provisioning image and update artefacts for the BitBox Base.
```

Comment:
Why change it back to BitBox Base?

Reply:
Meh, did not spot it when rebasing, thanks for finding this.